src/main/java/org/minions/devfund/reinaldo/WhackAMole.java

```diff
@@ -0,0 +1,126 @@
+package org.minions.devfund.reinaldo;
+
+/**
+ * This classes handle all function WhackAMoleTest.
+ */
+public class WhackAMole {
+
+    public static final int SCORE = 10;
+    private int attemptsLeft;
+    private int gridDimension;
+    private int score;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "score" to prevent any misunderstanding/clash with field "SCORE" defined on line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)